gui/preload-main.js

```diff
@@ -982,8 +983,10 @@ document.getElementById('hamlib_rigctld_stop').addEventListener('click', () => {
 
     // startPing button clicked
     document.getElementById("sendPing").addEventListener("click", () => {
-        var dxcallsign = document.getElementById("dxCall").value;
-        dxcallsign = dxcallsign.toUpperCase();
+        var dxcallsign = document.getElementById("dxCall").value.toUpperCase();
+        if (dxcallsign == "" || dxcallsign == null || dxcallsign == undefined)
+            return;
+        pauseButton(document.getElementById("sendPing"),500);
```

Comment:
I would increasing this to 1000 or maybe 2000.

Reply:
I'll change to 2000